internal/impl/devbox.go

```diff
@@ -727,6 +729,11 @@ func (d *Devbox) computeNixEnv() ([]string, error) {
 	}
 	envPairs = append(envPairs, pluginEnv...)
 
+	if featureflag.EnvConfig.Enabled() {
```

Comment:
can we move this into `computeNixEnv`?  That way both devbox run and shell will get it, not just shell.  It also helps us guarantee that these two paths behave as identically as possible.

Reply:
This is already in `computeNixEnv()` isn't it?